ui/src/app/layout/topics/topicmetadatatable/topic-metadata-table.component.html

```diff
@@ -265,3 +289,28 @@ <h4 class="modal-title">{{ 'Confirm Unsubscription' | translate }}</h4>
         </button>
     </div>
 </ng-template>
+
+<ng-template #deleteProducerDlg let-dlg>
+    <div class="modal-header">
+        <h4 class="modal-title">{{ 'Confirm Producer deletion' | translate }}</h4>
+        <button type="button" class="close" aria-label="Close" (click)="dlg.dismiss()">
+            <span aria-hidden="true">&times;</span>
+        </button>
+    </div>
+    <div class="modal-body">
+        <div class="mt-3">
+            <ngb-alert type="danger" [innerHTML]="'Producer wirklich löschen?' | translate" [dismissible]="false">
```

Comment:
Text should always be english here when using `| translate`. Also, as this is more than one word, or one name, should be a constant like DELETE_PRODUCER_PROMPT. The name of the producer should also be shown here, so user is sure they clicked the correct one.